draft/2024-05-01-this-week-in-rust.md

```diff
@@ -47,7 +47,9 @@ and just ask the editors to select the category.
 
 ## Crate of the Week
 
-<!-- COTW goes here -->
+This week's crate is [efs](Chttps://codeberg.org/RatCornu/efs), a no-std ext2 filesystem implementation with plans to add other file systems in the future.
```

Comment:
Thanks for putting this up @llogiq ! Been consuming TWiR for a while now and appreciate the hard work of those who make it happen each week.

Looks like there was a small typo in the URL for the CotW here:

```suggestion
This week's crate is [efs](https://codeberg.org/RatCornu/efs), a no-std ext2 filesystem implementation with plans to add other file systems in the future.
```

Reply:
Follow up question: why did the `inspect_links` step not fail in the `Run-Markdown-Checks` workflow?